common/persistence/nosql/nosqlplugin/cassandra/tasks.go

```diff
@@ -387,23 +387,25 @@ PopulateTasks:
 			continue
 		}
 
-		// Extract the TTL value
-		ttlValue, ttlExists := task["ttl"]
-
-		// Check if TTL is null or an integer
-		var ttl *int
-		if ttlExists && ttlValue != nil {
-			if ttlInt, ok := ttlValue.(int); ok {
-				ttl = &ttlInt // TTL is an integer
-			}
-		}
+		// TODO: no usage of ttl
```

Comment:
Let's leave it as-is. Because this is not the purpose of this PR. 
If there's something wrong, it would be easier to find root cause.